Equilobe.DailyReport.SL/DbService.cs

```diff
@@ -55,5 +58,27 @@ public static string GetReportTime(string baseUrl, long projectId)
                 .Select(qr => qr.ReportTime)
                 .FirstOrDefault();
         }
+
+        public static void SetExecutionDate(long id)
```

Comment:
Alt Nume pentru serviciu. Nu DbService. DataService in cel mai rau caz.

De asemenea daca e doar un DataService ar trebui sa stea in DataAccessLayer, nu in ServiceLayer